AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-staff-member/add-staff-member.component.html

```diff
@@ -1,6 +1,7 @@
 <form class="govuk-form-group" [formGroup]="form">
   <app-search-email
     [disabled]="emailDisabled"
+    locator="{{ staffMemberLocator }}"
```

Comment:
Change this to [locator]="staffMemberLocator"

Reply:
Updated